relay-server/src/metrics_extraction/transactions/mod.rs

```diff
@@ -55,11 +59,7 @@ fn extract_geo_country_code(event: &Event) -> Option<String> {
     geo.country_code.value().cloned()
 }
 
-fn is_low_cardinality(source: Option<&TransactionSource>) -> bool {
-    // `None` is used to mark a legacy SDK that does not send the transaction name,
-    // and we assume sends high-cardinality data. See `is_high_cardinality_transaction`.
```

Comment:
Should we restore this comment somewhere?

Reply:
I thought about this, too. Since this just references the other function, and that function is pretty clear on what it does, I chose to remove it entirely.